src/api/java/com/minecolonies/api/colony/managers/interfaces/ICitizenManager.java

```diff
@@ -29,14 +29,12 @@
 
     /**
      * Sends packages to update the citizens.
-     *  @param oldSubscribers    the existing subscribers.
-     * @param hasNewSubscribers the new subscribers.
-     * @param subscribers all subscribers
+     *  @param closeSubscribers    the existing subscribers.
+     * @param newSubscribers new subscribers
      */
     void sendPackets(
-            @NotNull final Set<EntityPlayerMP> oldSubscribers,
-            final boolean hasNewSubscribers,
-            @NotNull final Set<EntityPlayerMP> subscribers);
+      @NotNull final Set<EntityPlayerMP> closeSubscribers,
```

Comment:
How does this include `Global Subscribers`?

Reply:
they're not included, those do not receive views

Reply:
Why not? Why do they exists in the first place then?

Reply:
We have two kind of people who need to be updated about the colony:
- People close or inside the colony
- People who are outside of the colony but online and should receive messages about certain events.

Reply:
globalSubscribers only get messages but don't need views of the colony, since they can't see the colony, so they only need messages.

Reply:
Then it still should be renamed......

Reply:
ye still thinking about a fitting name :D